cssdb.json

```
@@ -504,6 +504,15 @@
       }
     ]
   },
+  {
+    "id": "overscroll-behavior",
```

Could we change this to include the `property` prefix? We do this for other properties.

```suggestion
    "id": "overscroll-behavior-property",
```